### ISSUE TITLE
Internal HTML viewer - ipv4/ipv6

### ISSUE THREAD

**Issue by staphon7** [oaubert/advene#81]
Hello there,  

My operating system is Arch Linux.  

**Here is the output of** `$ advene --version`**:**  
==> Advene development version release/3.13.1-1-gf2abf4d7

**How to reproduce the issue:**
- Have a firewall rule that refuses ipv6.
- Launch Advene with `$ ./bin/advene /<path_to>/Nosfertu_v14.azp`.
- Drag the button that says "Open a comment view in the browser" to the east of the video.
- Click on the "Cast" link.
- Then click on "Jonathon Harker" link.

**Actual result:**
- Advene is blocked.
- We have to kill the process via the appropriate binary/tool - kill, htop...
- Without ipv6 "support", I can see that the ipv6 address "::1" - which is ipv6 localhost - is asked only when I click on "Jonathon Harker" link.
- Here is the output - the interesting part - of `$ ss -n -taup`:

```
Netid State    Recv-Q Send-Q Local Address:Port Peer Address:Port  Process

tcp   SYN-SENT 0      1      [::1]:37448        [::1]:1234         users:(("python3",pid=3550186,fd=57))
```
- With ipv6 "support", everything is ipv4. At least, this is what I can deduce from ss command output.

I understand that this kind of firewall rule could be rare. I'm aware of my firewall rules. It was easy for me to check and then make a simple firewall rule to fix the issue.

Thank you

**Comment by oaubert** [comment]
What I do not understand is that you manage to visualize the pages (home, cast, etc). So the webserver is accessible. The only thing that changes when you click on the Jonathan Harker link is that is is a special link (http://localhost:1234/media/play/147533/150573 ) that plays the corresponding position in the Advene video player (more specifically it queues a "play" action and returns a "No content" response to the browser). So I do not see why it blocks the whole Advene process.  FYI the method that is called is https://github.com/oaubert/advene/blob/master/lib/advene/core/webcherry.py#L536

**Comment by staphon7** [comment]
Yes, you're right, the webserver is accessible, and I can notice it's not a problem to reload the page.
I would say it's not an issue anymore to me.
Thanks for your patience.
